app/templates/modal.html

```diff
@@ -4,12 +4,12 @@ <h3 class="modal-title">Create Your Own Chat Room!</h3>
     </div>
         <div class="modal-body">
             <form>
-                 Name that chat!<input type="text">
-             </form>
+                <h4>Name that chat:</h4><input id="newChatRoomName" type="text" ng-model="text" placeholder="What's it gonna be?">
+            </form><button class="btn" ng-click="$ctrl.add()"><i class="icon-plus"></i>Add</button>
```

Comment:
finally, we need to pass the value from our input to the `ok` function call here.
``` html
</form>
  <h4>Name that chat:</h4><input id="newChatRoomName" type="text" ng-model="text" placeholder="What's it gonna be?">
</form><button class="btn" ng-click="$ctrl.add(newChatRoomName)"><i class="icon-plus"></i>Add</button>
```
Notice how I'm passing the variable name `newChatRoomName` as an argument to the ng-click directive